src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,21 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
+            if (item.size > archiveByteLimit) {
                 return new Error(Strings.DND_MAX_FILE_SIZE_EXCEEDED);
+            } 
```

Comment:
Please move your `else if` to the same line as your closing `}`, should be `} else if {`